app/gui/enso-profiler-enso-data/src/beanpole.rs

```diff
@@ -0,0 +1,162 @@
+//! A diagram type for illustrating timings of messages between processes.
```

Comment:
I suspect that "beanpole" is a kind of a diagram. If so, could you link here wikipedia / other source explaining it to the reader, please?

Reply:
Added some description.